### PR TITLE
Create registerGJAccount.md

### PR DESCRIPTION
**Pull request by Mel0n7**
Documented the htttps://boomlings.com/database/account/registerGJAccount.php endpoint

### CODE REVIEW

File: docs/endpoints/account/registerGJAccount.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+```
+1
+```
+
+**Failed Request**
+
+| Error Code | Meaning |
+| --- | --- |
+| -2 | Already taken |
+| -4 | Longer than 20 characters |
```
Comment (Wyliemaster) [inline docs/endpoints/account/registerGJAccount.md]: There are a couple more error codes.  

Looking at the enum, these are the codes i expect to appear upon getting these errors however, this has not been checked

```cpp
enum AccountError
{
    kAccountErrorGeneric = -1,
    kAccountErrorTakenUsername = -2,
    kAccountErrorTakenEmail = -3,
    kAccountErrorInvalidUsername = -4,
    kAccountErrorInvalidPassword = -5,
    kAccountErrorInvalidEmail = -6,
    kAccountErrorPasswordTooShort = -8,
    kAccountErrorUsernameTooShort = -9,
};```

File: docs/endpoints/account/registerGJAccount.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+> This endpoint is used to register an account
+
+## Parameters
+
+| Parameter | Explanation | Optional |
+| --- | --- | --- |
+| `userName` | The username of the account to create | `False` |
+| `password` | The password of the account to create | `False` |
+| `email` | The email of the account to create | `False` |
+| `secret` | Wmfv3899gc9 | `False` |
```
Comment (Wyliemaster) [inline docs/endpoints/account/registerGJAccount.md]: Just a nit pick but for consistency between other files, this should probably specify the type of secret

| Parameter | Explanation | Optional |
| --- | --- | --- |
| `secret` | Account Secret: `Wmfv3899gc9 ` | `False` |

**Review by Wyliemaster — CHANGES_REQUESTED** [review]
Sorry to take so long, i've been quite busy so i didn't see the request until now. Here is some feedback

**Comment by Wyliemaster** [comment]
One thing i forgot to mention, would be appreciated if you could add the path to this in the accounts section of `_sidebar.md`

image

**Comment by Mel0n7** [comment]
Added everything you wanted!

**Pull request merged**